superset/examples/configs/datasets/examples/FCC_2018_Survey.yaml

```diff
@@ -930,7 +930,7 @@ columns:
   expression: null
   description: null
   python_date_format: null
-- column_name: rsrc_pluralsght
```

Comment:
@betodealmeida would the changes in this file pose any potential danger to anyone's workflows? Legit spelling fixes, but want to make sure this yaml change doesn't cause fallout.

Reply:
Just checked it's the column name on the examples, will prob cause some impact